cli.md

```diff
@@ -32,14 +34,25 @@ Every Horizon server requires a RethinkDB server to connect to. Use the `--conne
 * `--serve-static [PATH]` Enable serving static files via HTTP(S). You can additionally specify the path from which static files will be served (default: `./dist`).
 * `--config PATH` Which [config file][config-file] to use. Default: `.hz/config.toml`
 * `--debug [yes|no]` Print additional debug output. Default: `no`
+* `--schema-file [PATH]` Use a given schema file for the database. If the specified schema conflicts with the existing database, a warning will be printed to the console and the option will be ignored. (Use `hz schema apply --force` to override this.)
 
 ### Network options
 
 * `--bind HOST, -b HOST` The host name or IP address that the Horizon server should listen on for incoming requests. Can be specified multiple times to bind to multiple addresses. Default: `localhost`
 * `--port PORT, -p PORT` The port number the Horizon server should listen on for incoming requests. Default: `8181`
-* `--connect HOST:PORT, -c HOST:PORT` The host and port of the RethinkDB server to connect to. Default: `localhost:28015`
+* `--connect HOST:PORT, -c HOST:PORT` The host and port of the RethinkDB server to connect to, or a `rethinkdb://` URI string (see [RethinkDB options][rdbopts] in the configuration file documentation for details). Default: `localhost:28015`
 * `--key-file PATH` The key file to use for the HTTPS server. Default: `./horizon-key.pem`
 * `--cert-file PATH` The certificate to use for the HTTPS server. Default: `./horizon-cert.pem`
+* `--rdb_timeout SECONDS`: timeout to make the connection to the RethinkDB cluster, in seconds. Default: 20
+
+[rdbopts]: /docs/configuration/#rdbopts
+
+The following options provide alternatives to specifying these values in the `--connect` string:
+
+* `--rdb_host HOSTNAME`
+* `--rdb_port PORT`
+* `--rdb_user USERNAME`
```

Comment:
I think the `rdb_user` and `rdb_password` options also need to be added to embed.md .